velox/exec/Operator.h

```diff
@@ -194,6 +195,18 @@ class Operator {
         int32_t id,
         const std::shared_ptr<const core::PlanNode>& node) = 0;
 
+    // Translates plan node to join bridge. Returns nullptr if the plan node
+    // cannot be handled by this factory.
+    virtual std::shared_ptr<JoinBridge> translate(
+        const std::shared_ptr<const core::PlanNode>& node) {
+      return nullptr;
+    };
+
+    virtual OperatorSupplier translateToOpSupplier(
```

Comment:
Would you document this method? As mentioned above, perhaps, rename to toOperatorSupplier for clarity.

Reply:
Fixed